specification/columns/contractedcost.md

```diff
@@ -0,0 +1,37 @@
+# Contracted Cost
+
+Contracted Cost represents the cost calculated by multiplying [*contracted unit price*](#glossary:contracted-unit-price) and the corresponding [Pricing Quantity](#pricingquantity). Contracted Cost is denominated in the [Billing Currency](#billingcurrency) and is commonly used for calculating savings based on negotiation activities, by comparing it with [List Cost](#listcost). If negotiated discounts are not applicable, the Contracted Cost defaults to the List Cost.
+
+The ContractedCost column MUST be present in the billing data and MUST NOT be null. This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat) requirements, and be denominated in the BillingCurrency. When ContractedUnitPrice is present and is not null, multiplying the ContractedUnitPrice by PricingQuantity MUST produce the ContractedCost.
+
+In cases where the ContractedUnitPrice is present and null, the following applies:
```

Comment:
we might want to pick this up in #366 but i dont think there is any scenario where this value should be NULL

Reply:
For instance, in case of Adjustments (bulk refunds) SkuPriceId is null and thus Unit Prices, Quantities, etc. as well.

Reply:
Discussed on 28.3.24, there is a conflict between  quantity and unit price requirements when sku price id is not present and the need to not introduce artificial variance when the 4 cost columns are not in sync... ie one column contains a value but others are NULL. we will follow up in this in #366